common/icons/clear.js

```diff
@@ -0,0 +1,5 @@
+const Clear = () => (
```

Comment:
Just realised that we now have three variable name conventions on the go – ClassCase, camelCase and underscored. Given that icons are components and we want to be consistent, should we go for ClassCase everywhere?

Reply:
Yip - my linter moaned at me, so I did this.